CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## **[Unreleased]**
 
+- [#1335](https://github.com/wasmerio/wasmer/pull/1335) Change mutability of `memory` for `const` in `wasmer_memory_data_length` in the C API
```

Comment:
Minor nit, I think "`memory` for `const`` should be "`memory` to `const`"

Reply:
Thanks!